source/mapping-rules/edit-rules.txt

```diff
@@ -18,30 +18,49 @@ effect the next time a sync job is run.
 About this Task
 ---------------
 
-Options available on the ``edit`` pane depend on the type of 
+Options available on the :guilabel:`edit` pane depend on the type of 
 mapping rule you are editing. For an overview of each mapping rule 
 option see:
 
 - :ref:`New Documents <rm-new-documents>`
-  
+
 - :ref:`Embedded Array <rm-embedded-array>`
 
 - :ref:`Embedded Documents <rm-embedded-documents>`
 
 Steps
 -----
 
-1. Open a project.
+#. From the :guilabel:`Mapping` screen, click a table or collection name
+   on the :guilabel:`schema model` pane or diagram view.
 
-#. Click the ``Pencil`` icon to open the ``Edit mapping`` pane. 
+#. On the :guilabel:`schema model` pane on the right-side of the screen, 
+   click the :icon-fa5:`pencil-alt` icon.
 
 #. Edit the mapping rule.
 
-   .. image:: /img/mapping-rules/mapping-rule-edit-overview-2.png
-      :figwidth: 450 px
-      :alt: Mapping Rule Edit Overview
+   You can edit the collection name, field names, and 
+   include or exclude a field from your sync job by clicking the 
+   :icon-fa5:`check-square` next to the field name.
+
+#. (Optional) You can define :ref:`calculated fields <rm-calculated-fields>`,
+   :ref:`customized id fields <rm-id-field-customizations>`, and 
```

Comment:
Also add reference between step 5. 6. on Create rules to MongoDB to see fields for field customization options.